script.js

```diff
@@ -86,4 +92,32 @@ window.addEventListener('load', () => {
       `;
     });
   }
-});
\ No newline at end of file
+});
+
+listLink.addEventListener('click', () => {
+  sectionBookList.classList.remove('display-none');
+  sectionNewBook.classList.add('display-none');
+  contactSection.classList.add('display-none');
+});
+newLink.addEventListener('click', () => {
+  sectionNewBook.classList.remove('display-none');
+  sectionBookList.classList.add('display-none');
+  contactSection.classList.add('display-none');
+});
+contactLink.addEventListener('click', () => {
+  contactSection.classList.remove('display-none');
+  sectionBookList.classList.add('display-none');
+  sectionNewBook.classList.add('display-none');
+});
```

Comment:
- [ ] Good job adding the navigation and using pure javascript vanilla to do so. However kindly ensure your design fills up the full view of the webpage so as to match the given design

Your design
![image](https://github.com/over-geek/Awesome-books/assets/81932478/fe7fe90b-2f71-48e5-bd5a-56a4ea5fc7fd)

Given design
![image](https://github.com/over-geek/Awesome-books/assets/81932478/3bedd68a-220e-4ac0-a2e2-c3770996ee46)

### Note to follow the spacing of the elements so as to match the layout of the given design. This should include margin from the either end of the the page for all elements :red_circle: